web/src/components/pages/contribution/success.tsx

```diff
@@ -153,12 +161,42 @@ class Success extends React.Component<Props, State> {
           </div>
         )}
 
-        <ContributeMoreButton>
-          {type === 'speak' ? <MicIcon /> : <PlayOutlineIcon />}
-          <Localized id="contribute-more" $count={SET_COUNT}>
-            <span />
-          </Localized>
-        </ContributeMoreButton>
+        {wizardFinished &&
+          recordFinished &&
+          !listenFinished && (
+            <div className="kiosk-card">
+              <Localized id="kiosk-record-ok">
+                <h3 />
+              </Localized>
+
+              <Localized id="kiosk-record-submit">
+                <LinkButton rounded to={URLS.LISTEN} />
+              </Localized>
+            </div>
+          )}
+
+        {wizardFinished &&
+          recordFinished &&
+          listenFinished && (
+            <div className="kiosk-card">
+              <Localized id="kiosk-listen-ok">
+                <h3 />
+              </Localized>
+
+              <Localized id="kiosk-listen-submit">
+                <LinkButton rounded to={URLS.KIOSK} />
+              </Localized>
+            </div>
+          )}
+
+        {!wizardFinished && (
```

Comment:
Does that still show in non-Kiosk mode then?

Reply:
It should, yes, though I'm not 100% sure we need to hide it ?

Reply:
Retesting, it's not that disturbing IMHO, so I'll leave it.